py3status/modules/xrandr.py

```diff
@@ -399,11 +402,15 @@ def xrandr(self):
         self._set_available_combinations()
         self._choose_what_to_display()
 
-        if self.fixed_width is True:
-            output = self._center(self.displayed)
+        if (len(self.available_combinations) < 2 and
+                self.hide_if_single_combination):
+            full_text = ''
```

Comment:
> The [{output}] thing is already in the formatter to hide values if we don't get any.

So there are two confusing things for me: 

1. We always have the value, it's not like we don't have it sometimes. In fact, we always have one `displayed` and multiple `available`. So if you just use `format = '[{output}]'` in the config, without this PR you will see the output, the square brackets will not hide anything, they will not make output empty.

2. If we keep `hide_if_single_combination` for now, the goal is to set empty string in `full_text` in order to hide this module.... so why exactly `full_text = ''` is bad? It is the simplest code which achieves this.

If you have the time, could you maybe just make a branch where you show me what changes you have in mind, when you say "use `safe_format` and `[{output}]` instead of passing along `''`"? 

I have a strong suspicion that I don't know some crucial part of `safe_format` functionality, which you know 🙂